impeller/renderer/render_pass.h

```diff
@@ -244,6 +244,7 @@ class RenderPass : public ResourceBinder {
   const ISize render_target_size_;
   const RenderTarget render_target_;
   std::vector<Command> commands_;
+  std::vector<BufferView> vertex_buffers_;
```

Comment:
We cannot guarantee these all come from the same buffers.

Reply:
We don't have to do it as part of this PR but that seems ideal, right?

Reply:
I don't think that is reasonable. that would require us to know how much to allocate before we've recorded a single command. Otherwise we'll have multiple buffers and/or interleved different kinds of buffers

Reply:
Yea, I was thinking having them interleaved but if they were all in the same buffer we could potentially reuse that buffer across frames.  Also while binding they will all be next to each other.  No action required, just spitballing and making sure I'm understanding.

Reply:
So this is onyl really a gles2 problem. If I do the gles3 style render pass we won't even need to store them, so I don't think we should worry about it beyond low hanging fruit.